auslib/test/test_db.py

```diff
@@ -1311,7 +1316,7 @@ def testValidateConditionsTelemetryRaisesError(self):
 
     @mock.patch("time.time", mock.MagicMock(return_value=200))
     def testInsertWithEnabledCondition(self):
-        what = {"fooid": 11, "foo": "i", "bar": "jjj", "data_version": 1, "when": 909000, "change_type": "insert"}
+        what = {"fooid": 11, "foo": "i", "bar": "jjj", "data_version": 1, "when": 909000, "change_type": "update"}
```

Comment:
Looks like change_type is not verified when the db is read back, by design or just missed when change_type came along ?